### ISSUE TITLE
[BUG] CallZome not working (timeout) in Windows

### ISSUE THREAD

**Issue by guillemcordoba** [holochain/holochain#3028]
**Describe the bug**

In Windows, the new holochain v0.2.3-rc.0 is giving timeouts on call_zome, without ever returning. I encountered it while running [gather](https://github.com/darksoil-studio/gather/releases/tag/v0.0.4) in windows. All other platforms work.

I then downloaded a happ in Windows and tried to run tryorama tests with it and that same holochain version, and it was also failing the tests and giving timeouts.

AdminWebsockets do work though.

**Expected behavior**

Holochain responds to `call_zome` requests on Windows.

**System information:**
 - OS: [e.g. iOS]: Windows 10
 - Holochain: v0.2.3-rc.0
  
**Steps to reproduce**

1. Download [gather](https://github.com/darksoil-studio/gather/releases/tag/v0.0.4)
2. Run it in windows
3. You will see a "call_zome timeout in 60000 ms error from @holochain/client"

**Comment by matthme** [comment]
Can confirm the same to happen in the launcher built for holochain 0.2.3-rc.0

**Comment by ThetaSinner** [comment]
Likewise, reproducible on the linked installer and when running on Launcher

**Comment by ThetaSinner** [comment]
I'm seeing lots of 

```
[2023-11-13T12:10:55.879814200+00:00] INFO - [HOLOCHAIN 0.2.3-rc.0] [2m2023-11-13T12:10:55.879737Z[0m [33m WARN[0m [2mkitsune_p2p::spawn::actor::space[0m[2m:[0m [2mC:\Users\runneradmin\.cargo\registry\src\index.crates.io-6f17d22bba15001f\kitsune_p2p-0.2.3-rc.0\src\spawn\actor\space.rs[0m[2m:[0m[2m1086:[0m delegate broadcast error [3merr[0m[2m=[0mKitsuneError(Other(Custom { kind: Other, error: Timeout }))

[2023-11-13T12:10:55.879856100+00:00] INFO - [HOLOCHAIN 0.2.3-rc.0] [2m2023-11-13T12:10:55.879737Z[0m [33m WARN[0m [2mkitsune_p2p::spawn::actor::space[0m[2m:[0m [2mC:\Users\runneradmin\.cargo\registry\src\index.crates.io-6f17d22bba15001f\kitsune_p2p-0.2.3-rc.0\src\spawn\actor\space.rs[0m[2m:[0m[2m1086:[0m delegate broadcast error [3merr[0m[2m=[0mKitsuneError(Other(Custom { kind: Other, error: Timeout }))

[2023-11-13T12:10:55.879862600+00:00] INFO - [HOLOCHAIN 0.2.3-rc.0] [2m2023-11-13T12:10:55.879735Z[0m [33m WARN[0m [2mkitsune_p2p::spawn::actor::space[0m[2m:[0m [2mC:\Users\runneradmin\.cargo\registry\src\index.crates.io-6f17d22bba15001f\kitsune_p2p-0.2.3-rc.0\src\spawn\actor\space.rs[0m[2m:[0m[2m1086:[0m delegate broadcast error [3merr[0m[2m=[0mKitsuneError(Other(Custom { kind: Other, error: Timeout }))

[2023-11-13T12:10:55.879866900+00:00] INFO - [HOLOCHAIN 0.2.3-rc.0] [2m2023-11-13T12:10:55.879752Z[0m [33m WARN[0m [2mkitsune_p2p::spawn::actor::space[0m[2m:[0m [2mC:\Users\runneradmin\.cargo\registry\src\index.crates.io-6f17d22bba15001f\kitsune_p2p-0.2.3-rc.0\src\spawn\actor\space.rs[0m[2m:[0m[2m1086:[0m delegate broadcast error [3merr[0m[2m=[0mKitsuneError(Other(Custom { kind: Other, error: Timeout }))

[2023-11-13T12:10:55.974194500+00:00] INFO - [HOLOCHAIN 0.2.3-rc.0] [2m2023-11-13T12:10:55.974121Z[0m [33m WARN[0m [2mkitsune_p2p::spawn::actor::space[0m[2m:[0m [2mC:\Users\runneradmin\.cargo\registry\src\index.crates.io-6f17d22bba15001f\kitsune_p2p-0.2.3-rc.0\src\spawn\actor\space.rs[0m[2m:[0m[2m1086:[0m delegate broadcast error [3merr[0m[2m=[0mKitsuneError(Other(Custom { kind: Other, error: Timeout }))
```

Which suggests that gossip with some other node isn't working correctly but that shouldn't cause zome calls to fail unless they need the network. Is that the case for the zome call that's failing?

**Comment by guillemcordoba** [comment]
Well unfortunately I don't have access to the logs but it was failing still when only one existed in the network... So no I don't think that's the issue?

**Comment by jost-s** [comment]
I've reproduced this problem under WSL with Holochain 0.2.3-rc.0 and also with 0.2.2. It's not a bug introduced by 0.2.3-rc.0. I'm going back through the versions to see since when this has been a problem.

**Comment by matthme** [comment]
0.2.3-beta-rc.1 worked fine for me on Windows when I tested this morning. I only got the zome call timeouts with 0.2.3-rc.0.

**Comment by jost-s** [comment]
mewsfeed under 0.2.3-rc.0 has the same problem, with the difference that I can create a profile and then subsequent zome calls time out.

**Comment by mjbrisebois** [comment]
If you can confirm that the request is being received by conductor, but it's not running the zome call (ie. a `debug!` line at the start of your function is not running), then this could be the same issue we ran into a couple of versions ago.

**Comment by jost-s** [comment]
Perplexing, I just run "gather" successfully on our hosted Windows machine. Windows 10, WSL 2, Holochain v0.2.3-rc.0. Runs perfectly fine.
<img width="2548" alt="image" src="https://github.com/holochain/holochain/assets/28270981/6990c3eb-db71-4c80-a393-d8e05fccf974">

Makes me think that it heavily depends on the machine's performance.

**Comment by ThetaSinner** [comment]
@mjbrisebois Could you point me at the issue you're referring to please?

@jost-s FWIW I have a powerful Windows machine that fails to run gather on every attempt. Sometimes I get a bit further and can create a profile then see the UI but that's as far as it gets for me. Doesn't mean that performance isn't relevant because I could still have different hardware that's part of the answer but I suspect it's only part of the answer.

**Comment by jost-s** [comment]
@ThetaSinner Are you running the standalone app or the dev version?

**Comment by mjbrisebois** [comment]
@ThetaSinner An issue was never made here on GitHub because it was only happening on the App Store DHT.  I don't think it could be the same issue though because this one is happening on tests that create new DHTs.

**Comment by ThetaSinner** [comment]
I'm running the webhapp from here https://github.com/darksoil-studio/gather/releases/tag/v0.0.4 in the 0.11.3 launcher

**Comment by jost-s** [comment]
Right, then please give the dev version (clone repo and run npm start) a shot too. I'm suspecting something buggy in the interplay between happ + environment (+ os).

**Comment by guillemcordoba** [comment]
Careful @jost-s and @ThetaSinner , the gather repo is still at holochain v0.2.2, since v0.2.3-rc.0 hasn't been made available in holonix yet. I compiled and built the gather installable version from [this repo](https://github.com/darksoil-studio/rostanga-test). Sorry if this has caused confusion, but the state of the published crates vs what we want to test makes it tricky for me too...

**Comment by ThetaSinner** [comment]
@guillemcordoba Understood. The app should be compatible with the rc I believe but I will bear that in mind.

There is a new Holonix version for the rc too if that's useful to you. There's now a `0_2_rc` version you can point at instead of `0_2` which has the cached release candidate under it :)

**Comment by jost-s** [comment]
@guillemcordoba How did you build the rc.0 dependencies like alerts into the test release?

**Comment by guillemcordoba** [comment]
I didn't, I'm using hdk v2.2.0 to build everything for the app, and then installing it in a v0.2.3-rc.0 conductor. This is part of what I was trying to say with the previous comment. As @ThetaSinner said, I thought it should be compatible, and it was when I tested in local, so I assumed there were no version incompatibilities, as semver v0.2.2 should be compatible with v0.2.3.

**Comment by jost-s** [comment]
Right, then simply using holochain-0.2.3-rc.0 in the flake should be fine for testing with the cloned repo.

**Comment by abe-njama** [comment]
@dellams we're still debugging the zome call timeouts happening with wasmer v4.2.2. The wasmer bump had to be reverted so we can get around this problem. 
This issue is actively happening in `0.2.3-rc.0` which had the newer wasmer version. `0.1.5` and `0.1.6` should be okay. They are running an older wasmer version.

**Comment by jost-s** [comment]
This bug should be fixed with #3208. Please try it out again and report back.

**Comment by guillemcordoba** [comment]
Hey! This particular error is not present anymore no, so I'll close this.
